buck2.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: buck2
+  version: 0.0_git20230410
+  epoch: 0
+  description: "Build system, successor to Buck"
+  copyright:
+    - license: MIT
+environment:
+  contents:
+    packages:
+      - libLLVM-15
+      - wolfi-base
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - rustup
+vars:
+  commit: 54decd83a2adf998ec9f1a52958d726924241f32
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/facebook/buck2
+      # They don't have any releases yet, so use a commit below.
+      branch: main
+  - name: Configure and build
+    runs: |
+      git checkout ${{vars.commit}}
```

Comment:
We should probably open an issue for this to be supported as a Melange pipeline or something

Reply:
They changed it up a bit so there's now a `latest` tag that gets bumped semi-regularly. It makes our side easier but is still very unusual.

Reply:
Interesting!